### DISCUSSION TITLE
Has anyone ever put their heart and soul into a repo, just to have their baby stolen from you like that one scene in Zorro?

### DISCUSSION THREAD

**Discussion by jivip** [community/community#42151] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I was working at this company for a year and a half. I developed a working CRM to ERP solution by myself with no help other than verbal outlines from those in charge. It was like a small scale SalesForce backend if you will, all for the sake of a legitimate full-time contract. 

I was hired under an internship contract last Summer and looking back, I believe I was coerced into working for said company in the hopes of getting a full-time contract after I graduated. Unfortunately, I was given the run around and recently got terminated after pointing out that hiring me for low-wages in the hopes of giving me a full-time contract was a Conflict of Interest by the engineering standards in Canada.

Little did I know that the engineering standards are governed by your respective province as opposed to a federally governing body.

I wanted to know if anyone had any thoughts, advice or stories they'd like to share.

**Comment by samharp** [comment, marked as answer]
While I have plenty of love for Zorro, I lack an appreciation for large companies not valuing employee attachment to projects (or not being entirely honest with them). I had a similar thing happen to me a couple of years ago where I designed a massive client-information portal for the company I was working at, only to have the plug pulled so "we can focus on other projects". Having the 'plug pulled' on multiple projects that I was passionate about ultimately led to me leaving the company (for several other reasons) as I couldn't take putting my heart and soul into a project just to have it lead nowhere.

I am in the United States, so I can't really speak on behalf of Canadian laws, but it is also a shame that they weren't more honest with you from the get-go (or even leading up to the termination). If they had been more up front with you, maybe things would have transitioned smoother.

I hope my story is somewhat helpful in showing that you aren't alone, but here's to hoping that your next job is a bit more of a positive experience.
